.github/workflows/wasm-automatic-release.yml

```diff
@@ -0,0 +1,34 @@
+name: WASM Automatic Release
+
+on:
+  pull_request:
+    branches:
+      - dev
+      - main
+    types: [closed]
+jobs:
+  call-create-release-workflow:
+    if: github.event.pull_request.merged == true
+    # owner/repository has to be static, see https://github.community/t/env-variables-in-uses/17466
+    uses: iotaledger/identity.rs/.github/workflows/shared-release.yml@dev
+    with:
+      changelog-config-path: ./bindings/wasm/.github_changelog_generator
+      dev-tag-regex: ^wasm-v[0-9]+\.[0-9]+\.[0-9]+-(dev)\.\d*$
+      main-tag-regex: ^wasm-v[0-9]+\.[0-9]+\.[0-9]+$
+      create-github-release: false
```

Comment:
Question: was it decided that we only create GitHub releases for the Rust library, not the bindings? I can't recall.

Reply:
This is the latest decision, yes.